packages/idempotency/tests/e2e/makeFunctionIdempotent.test.FunctionCode.ts

```diff
@@ -0,0 +1,67 @@
+import { DynamoDBPersistenceLayer } from '../../src/persistence';
+import { makeFunctionIdempotent } from '../../src/makeFunctionIdempotent';
+import { Logger } from '@aws-lambda-powertools/logger';
+import { Context } from 'aws-lambda';
+
+const IDEMPOTENCY_TABLE_NAME = process.env.IDEMPOTENCY_TABLE_NAME;
```

Comment:
```suggestion
const IDEMPOTENCY_TABLE_NAME = process.env.IDEMPOTENCY_TABLE_NAME || 'table_name';
```

Reply:
Why should we add default?

Reply:
TypeScript is complaining because environment variables can be `string | undefined`. We know it's there because we created the function, but it doesn't know it.

To make it shut up we have three options:
- Add a default (we do this in all other e2e tests) so it's always a string
- Change the declaration to `process.env.IDEMPOTENCY_TABLE_NAME!` - note the exclamation mark which tells TS this value is for sure there
- Introduce some `if` statement that check whether the variable it's defined or not, and if not throw an error

Number three is verbose, number 2 would generate a warning from eslint which we'd have to suppress, so we are left with number one.

Reply:
Interesting, I don't have the warning :) I will add a default value.

Reply:
![image](https://github.com/awslabs/aws-lambda-powertools-typescript/assets/7353869/5cce6ca3-c949-4c2d-878d-c4e22b856db8)

I hope that after applying the linting/formatting repo-wide we'll have the same configs & see same warnings.